.evergreen.yml

```diff
@@ -305,6 +305,52 @@ functions:
 
           DEBUG=${debug} MONGODB_VERSION=${mongodb_version|4} npm run --unsafe-perm --workspace compass-e2e-tests test-packaged-ci
 
+  test-connectivity:
+    - command: shell.exec
+      # Fail the task if it's idle for 10 mins
+      timeout_secs: 600
+      params:
+        working_dir: src
+        shell: bash
+        env:
+          <<: *compass-env
+        script: |
+          set -e
+          # Load environment variables
+          eval $(.evergreen/print-compass-env.sh)
+
+          git clone https://github.com/mongodb-js/devtools-docker-test-envs.git test-envs
+          cd test-envs
+          git checkout v1.2.4
+          cd ..
+          docker-compose -f test-envs/docker/enterprise/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/ldap/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/scram/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/sharded/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/ssh/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/tls/docker-compose.yaml up -d
+          docker-compose -f test-envs/docker/kerberos/docker-compose.yaml up -d
+
+          stop_all_docker_containers() {
+            docker-compose -f test-envs/docker/enterprise/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/ldap/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/scram/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/sharded/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/ssh/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/tls/docker-compose.yaml down -v --remove-orphans
+            docker-compose -f test-envs/docker/kerberos/docker-compose.yaml down -v --remove-orphans
+          }
+
+          trap "stop_all_docker_containers" ERR
+
+          DEBUG=${debug} \
+            MONGODB_VERSION=${mongodb_version|4} \
+            SKIP_KERBEROS_TESTS=true \
+            COMPASS_RUN_DOCKER_TESTS=true \
```

Comment:
Can you move these to the `env` key for consistency with other steps?

Reply:
Sure